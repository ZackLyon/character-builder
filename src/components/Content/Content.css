.counts {
  margin-bottom: 1vh;
}
