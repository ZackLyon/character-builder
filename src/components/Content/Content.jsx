import React from 'react';
import './Content.css';

export default function Content({
  catchPhrases,
  headCount,
  middleCount,
  legCount,
}) {
  return (
    <div className='content'>
      <div className='counts'>
        <div>The head has been changed: {headCount} times.</div>
        <div>The middle has been changed: {middleCount} times.</div>
        <div>The legs have been changed: {legCount} times.</div>
      </div>
      Check out my snappy catch phrases:
      {catchPhrases.map((phrase) => (
        <div>"{phrase}"</div>
      ))}
    </div>
  );
}
