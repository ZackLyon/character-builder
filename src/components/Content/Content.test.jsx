import { render } from '@testing-library/react';
import Content from './Content';

it('should display the content on the screen', () => {
  const { container } = render(
    <Content
      catchPhrases={['something', 'something else']}
      headCount={3}
      middleCount={2}
      legCount={5}
    />
  );

  expect(container).toMatchSnapshot();
});
