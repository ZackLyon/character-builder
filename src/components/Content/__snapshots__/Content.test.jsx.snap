// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the content on the screen 1`] = `
<div>
  <div
    class="content"
  >
    <div
      class="counts"
    >
      <div>
        The head has been changed: 
        3
         times.
      </div>
      <div>
        The middle has been changed: 
        2
         times.
      </div>
      <div>
        The legs have been changed: 
        5
         times.
      </div>
    </div>
    Check out my snappy catch phrases:
    <div>
      "
      something
      "
    </div>
    <div>
      "
      something else
      "
    </div>
  </div>
</div>
`;
